error handling for unsupported manager

`$ wtd npm2 express`

```bash
module.js:442
    throw err;
    ^

Error: Cannot find module './lib/parser/npm2'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/hanksudo/Github/wtd/src-es5/cli.js:29:16)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
```

--- a/src/cli.js
+++ b/src/cli.js
@@ -29,13 +29,21 @@ if (!manager) {
 } else if (!name) {
   console.log('❌ ERROR: package name is required')
 } else {
-  const parser = require(`./lib/parser/${manager}`)
-  const result = parser(name, { listUrl })
+  try {
+    const parser = require(`./lib/parser/${manager}`)
+    const result = parser(name, { listDev, listUrl })
 
-  result.then(jsonAry => {
-    jsonAry.map(json => {
-      const { name, description } = json
-      console.log(`${name} - ${description}`)
+    result.then(jsonAry => {
+      jsonAry.map(json => {
+        const { name, description } = json
+        console.log(`${name} - ${description}`)
+      })
     })
-  })
+  } catch (e) {
+    if (e.code === 'MODULE_NOT_FOUND') {
+      console.log('❌ ERROR: package manager is not found')
+    } else {
+      throw e
+    }
+  }
 }

